serialization/src/main/java/net/corda/v5/serialization/SerializedBytes.java

```diff
@@ -10,10 +10,18 @@
 @SuppressWarnings("unused")
 @CordaSerializable
 public final class SerializedBytes<T> extends OpaqueBytes {
+    /**
+     * Constructs a SerializedBytes holding the specified bytes
+     * @param bytes byte array holding serialized form of type T
+     */
     public SerializedBytes(@NotNull byte[] bytes) {
         super(bytes);
     }
 
+    /**
+     * Returns a text description of this SerializedBytes object
```

Comment:
needed?

Reply:
yes, it's the method description/return value description again